### ISSUE TITLE
Issue regarding offline execution

### ISSUE THREAD

**Issue by ykitanob** [NCGM-genome/WGSpipeline#11]
Dear Support Team,
I encountered an issue while trying to run  cwltool on our server. 
Our server is not connected to the internet, and when I tried to run the Workflow, I received an error message below.
```
    raise JavascriptException(
cwl_utils.errors.JavascriptException: Long-running script killed after 60 seconds: Javascript expression was: (1)
stdout was:
stderr was: FATAL:   Unable to handle docker://docker.io/node:alpine uri: failed to get checksum for docker://docker.io/node:alpine: error pinging docker registry registry-1.docker.io: Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: Temporary failu
```

 To get around this issue, I had to add the ```--disable-js-validation``` option.
I am not sure the correct way to handle this situation.
Could you please provide some advice or recommendations on how to perform validation without connecting to the internet?

**Comment by hacchy1983** [comment]
Hi @ykitanob 
To run our workflow on an offline server, the following steps are required.
1/ Create a singularity image on another computer that can connect to the Internet by running the workflow once. Then the `*.sif` file(s) will be created. Note that the difference in OS and Singularity version between offline and online servers will not be a problem in most cases. 
2/ Upload the `*.sif` file(s) to the offline server and locate the file(s) in the working directory. 

Could you try these steps?

Best regards, 
Tsuyoshi

**Comment by ykitanob** [comment]
Thank you for your response. I wanted to provide some additional information as there were some omissions in my previous message.

Regarding your suggestions 1 and 2, we have already taken care of them.
We created the .sif file in an internet environment and placed it in the working directory. 
We were able to run all three steps of the tutorial by adding the ```--disable-js-validation``` option. However, without this option, cwltool performs validation and a JavascriptException occurs.

Thank you again for your help.
